sockeye/inference.py

```diff
@@ -532,9 +532,15 @@ def _beam_search(self,
                      source: mx.nd.NDArray,
                      source_length: mx.nd.NDArray,
                      bucket_key: int,
-                     max_output_length: int) -> Tuple[List[int], np.ndarray, float]:
+                     max_output_length: int) -> Tuple[List[List[int]], List[np.ndarray], mx.nd.NDArray]:
         """
-        Translates a single sentence using beam search.
+            Translates a single sentence using beam search.
```

Comment:
unnecessary indentation. Also, please add a newline between the docstring and the ":param" lines so that sphinx can generate well-formed documentation.